app/assets/javascripts/application.js

```diff
@@ -16,7 +16,17 @@
 //= require bootstrap-sprockets
 //= require bootstrap
 //= require rails-ujs
+//= require elemental
 //= require cocoon
 //= require activestorage
 //= require turbolinks
 //= require_tree .
+
+$(document).on('change', '*[data-behavior="TutorTemple.experienceCheckbox"]', function (e) {
+  input = $(e.target).parents('.nested-fields').find('.end_date');
+  if (e.target.checked) {
+    input.prop('disabled', true);
+  } else {
+    input.prop('disabled', false);
+  }
+});
```

Comment:
why do we need `elemental` if you don't even use it? Let's also use coffee (`application.js` -> `application.js.coffee`)
This all can be simplified to:
```
$(document).on 'change', '.current_workplace_checkbox', ->
  $(@).parents('.nested-fields').find('select.end_date').prop('disabled', @.checked)
```
If suggestion in comment below applied